crates/rooch/src/commands/account/commands/verify.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use async_trait::async_trait;
+use clap::Parser;
+use rooch_types::{
+    error::{RoochError, RoochResult},
+    framework::auth_payload::AuthPayload,
+};
+
+/// Verify an auth payload hex
+#[derive(Debug, Parser)]
+pub struct VerifyCommand {
+    /// an auth payload hex for verify
+    #[clap(long)]
+    input: String,
```

Comment:
Input should be signature

Reply:
> Input should be signature

Added signature and message hash for verification.